src/tsconfig-base.json

```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-        "lib": ["es5"],

```

Comment:
We actually do use dom and scripthost types in `sys.ts`. Only thought this worked because gulp was ignoring the inheritance.